docs/source/overview/prerequisites.rst

```diff
@@ -63,14 +63,24 @@ An example is the following:
 
 ::
 
-  ev_op_max_order: 10
+  configs:
```

Comment:
I guess that the elements after should be indented, isn't it?

Reply:
[c063287](https://github.com/NNPDF/pineko/pull/98/commits/c063287b0a2e562b58e2d4f0900bcdb0ff828b2f)